.spellcheck.yml

```diff
@@ -0,0 +1 @@
+source_files: docs/**/*.md docs/**/*.mdx
```

Comment:
why do you need to put `source_files` both in the `.spellcheck.yml` and also in the `workflows/spellcheck.yml` ?

I tihnk one is enough no? Otherwise it is confusing.